src/i2c_device.c

```diff
@@ -87,6 +87,7 @@ void Init_TC3587(uint8_t fmt) {
 #ifdef _DEBUG_TC3587
     debugf("\r\nDetecte TC3587...");
 #endif
+#if (0) // some vtx cannot pass tc3587 detect, I don't know the reason.
```

Comment:
Is it safe to ignore the code in all cases? Would it be better to add a timeout or something?

Reply:
Yes.The original intention of this code is to detect whether the tc3587 is detected normally, but after adding it, the tc3587 of some VTX starts normally.
There is no problem without it. It is not included in firmware version 06222022.